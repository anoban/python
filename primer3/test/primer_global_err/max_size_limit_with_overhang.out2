../src/primer3_core: PRIMER_MAX_SIZE + SEQUENCE_OVERHANG_LEFT exceeds built-in maximum of 60
