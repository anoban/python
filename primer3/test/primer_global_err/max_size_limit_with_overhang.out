PRIMER_TASK=check_primers
SEQUENCE_TEMPLATE=TGCTTTCAATCATGTGGGTGATTTTATAGATTGTCATCACCATAGTAACTGTTTTAACTCACGTCCCCTGTGTGAGAACGCCATGGGCTGTCGGGCTTCGCCAACTTGCTCTCATTTCTA
SEQUENCE_PRIMER=TGCTTTCAATCATGTGGGTGA
SEQUENCE_OVERHANG_LEFT=TCACCCACATGATTGAAATTTCACCCACATGATTGAAATT
P3_COMMENT=the addition of overhangs violates primer size limits
PRIMER_ERROR=PRIMER_MAX_SIZE + SEQUENCE_OVERHANG_LEFT exceeds built-in maximum of 60
=
